hazelcast/src/main/java/com/hazelcast/jet/Jet.java

```diff
@@ -27,6 +35,7 @@
  *
  * @since 3.0
  */
+@Deprecated
```

Comment:
Each deprecated class/method should have a javadoc pointing the user to the replacement. If the new method fully replaces the deprecated one, then we should also remove the entire javadoc from the old method.